js/views/pageNavVw.js

```diff
@@ -501,11 +501,13 @@ module.exports = baseVw.extend({
   },
 
   navBackClick: function(){
-    window.history.back();
+    history.action = 'back';
```

Comment:
This also doesn't take into account the fact that in some areas we are navigating via Backbone.history.navigate and passing in a 'replace: true' option, which replaces the current route rather than adding it to the history queue. This will be another scenario where your 'position' calculation will be off, since it shouldn't be incremented if we are replacing the route.